apps/web/src/components/mcp-server-list-dialog.tsx

```
@@ -0,0 +1,436 @@
+              checked={enableWebSearch}
+              onCheckedChange={() => setEnableWebSearch(!enableWebSearch)}
+            />
+          </div>
+        </AccordionTrigger>
+        <AccordionContent className="px-3">
+          <div className="grid grid-cols-1 auto-cols-min gap-2">
+            <p className="text-muted-foreground">
+              Get real-time web search results.
+              <a
+                href="https://openrouter.ai/docs/features/web-search"
+                target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)